x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_management/stream_systems/detected_system_events.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { System } from '@kbn/streams-schema';
+
+import React from 'react';
+import { Chart, BarSeries, Settings } from '@elastic/charts';
+import { useElasticChartsTheme } from '@kbn/charts-theme';
+import { EuiButtonEmpty, EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useSystemEvents } from './hooks/use_system_events';
+
+export const DetectedSystemEvents = ({ system }: { system: System }) => {
```

Comment:
The name `DetectedSystemEvents` throws me off. I think `detection` is more applicable in the context of sig events. For a system, it's just "data for that system". Maybe just something like <SystemThroughputChart/>?